doc/sphinx-guides/source/installation/advanced.rst

```diff
@@ -35,3 +35,48 @@ If you have successfully installed multiple app servers behind a load balancer y
 You would repeat the steps above for all of your app servers. If users seem to be having a problem with a particular server, you can ask them to visit https://dataverse.example.edu/host.txt and let you know what they see there (e.g. "server1.example.edu") to help you know which server to troubleshoot.
 
 Please note that :ref:`network-ports` under the Configuration section has more information on fronting your app server with Apache. The :doc:`shibboleth` section talks about the use of ``ProxyPassMatch``.
+
+Optional Components
+-------------------
+
+Standalone "Zipper" Service Tool
+++++++++++++++++++++++++++++++++
+
+As of Dataverse v5.0 we offer an experimental optimization for the
+multi-file, download-as-zip functionality. If this option
+(``:CustomZipDownloadServiceUrl``) is enabled, instead of enforcing
+the size limit on multi-file zipped downloads (as normally specified
+by the option ``:ZipDownloadLimit``), we attempt to serve all the
+files that the user requested (that they are authorized to download),
+but the request is redirected to a standalone zipper service running
+as a cgi-bin executable under Apache. Thus moving these potentially
+long-running jobs completely outside the Application Server (Payara);
+and preventing worker threads from becoming locked serving them. Since
+zipping is also a CPU-intensive task, it is possible to have this
+service running on a different host system, freeing the cycles on the
+main Application Server. (The system running the service needs to have
+access to the database as well as to the storage filesystem, and/or S3
+bucket).
+
+Please consult the scripts/zipdownload/README.md in the Dataverse 5
+source tree for more information. 
+
+To install: You can follow the instructions in the file above to build
+``ZipDownloadService-v1.0.0.jar``. It will also be available, pre-built as part of the Dataverse release on GitHub. Copy it, together with the shell
+script scripts/zipdownload/cgi-bin/zipdownload to the cgi-bin
+directory of the chosen Apache server (/var/www/cgi-bin standard).
+You may need to make extra Apache configuration changes to make sure /cgi-bin/zipdownload is accessible from the outside.
+For example, if this is the same Apache that's in front of your Dataverse Payara instance, you will need to add another pass through statement to your configuration:
+
+``ProxyPassMatch ^/cgi-bin/zipdownload !``
+
+Edit the config lines in the shell script (zipdownload) to configure
+database access credentials. Do note that the executable does not need
+access to the entire Dataverse database. A security-conscious admin
+can create a dedicated database user with access to just one table:
+``CUSTOMZIPSERVICEREQUEST``.
+
+to activate in Dataverse::
```

Comment:
@landreev Sorry to drop $0.02 of code review in here while it's in QA... 

- this sentence should start with an uppercase "To..." 
- there only needs to be one colon `:` after "...Dataverse:"
- the curl command in the next line, below is missing the double-`` formatting -- see curl commands examples you included in config.rst in this same PR.

Reply:
OK, I capitalized the "To". 
The other 2 things you mentioned are legit though: 
the double colon ("::"), followed by some TAB-shifted text produces ONE colon, and formats the text as fixed-width, similar to double-backtick. 
As can be seen here:
https://github.com/IQSS/dataverse/blob/6505-optimize-zip-downloads/doc/sphinx-guides/source/installation/advanced.rst

Reply:
Ahh. So it does. I managed to dig into other rst files to find more examples and confirm it properly formats the command snippets. When I was writing my review, I was only eyeballing it in the PR change log and have not used that formatting method myself.